packages/core/src/external/surescripts/codes.ts

```diff
@@ -37,7 +37,23 @@ export const PLAN_CODE_NAME: Record<PlanCode, PlanCodeName> = {
   "06": "Workers' Compensation",
 };
 
-export const PAYMENT_CODES = ["01", "02", "03", "04", "05", "06", "07", "99"] as const;
+export const PAYMENT_CODES = [
```

Comment:
This was required to handle a schema change on Surescript's end - the specification clearly states that the values would be "01", "02", etc, and the initial data we received before this month always had the trailing zero. This is therefore a backwards-compatible change to account for both allowed enum types, and defines a function for converting to the payment code name with either format.